x/migration/module/autocli.go

```diff
@@ -32,6 +38,13 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					RpcMethod: "UpdateParams",
 					Skip:      true, // skipped because authority gated
 				},
+				{
+					RpcMethod:      "CreateMorseAccountState",
+					Use:            "create-morse-account-state [accounts]",
+					Short:          "Create morse_account_state",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "accounts"}},
```

Comment:
What are `accounts` used for? Shouldn't it be `morse_account_state` + `morse_account_state_hash`?

Reply:
I'll comment this out because it doesn't make sense - the important part is the `skip: true` below.